src/routes/safe/components/Apps/appsList.js

```diff
@@ -38,3 +38,7 @@ export default [
     },
   },
 ]
+
+export default appList
+
+export const getAppInfo = (appName: string) => appList.find(app => app.name === appName)
```

Comment:
just curious, isn't it better to search by id instead of name?

Reply:
it was just a test, I need to refactor it to persist the appId. I'm workign on that on issue #501